cmd/glasskube/cmd/describe.go

```diff
@@ -329,6 +331,17 @@ func status(pkg ctrlpkg.Package) string {
 	}
 }
 
+func message(pkg ctrlpkg.Package) string {
+	pkgStatus := client.GetStatusOrPending(pkg)
```

Comment:
The `status` function is also getting the status by calling this method, so I think at this point it makes sense to get the `pkgStatus` in the calling function, and then pass the status object into `status` and `message`.

Reply:
can you elaborate a bit on the expected functionality here ?

Reply:
`pkgStatus := client.GetStatusOrPending(pkg)` should be done before line 138, and then `pkgStatus` should be passed into `status()` and `message()`. This way it gets evaluated only once, and can be passed into the functions.

Reply:
@christophenne added new changes let me know